### PR TITLE
[Site-310] Adjust highlight post card

### PR DESCRIPTION
**Pull request by nenispng**
This branch name does not match the task name. Sorry :(

**Link to issue 🔗**
https://labcodes.atlassian.net/browse/SITE-310

**What's new? ✨**
- Deleted some deprecated color variables
- Fixed image and content position and size on highlight card
- Brand-new highlight post card on mobile viewports
- Hover and focus effects for all post cards
- Fixed display issue on wrapper div
- General file organization to make some components work anywhere

### CODE REVIEW

File: themes/medius/scss/home.scss [hunk 1]
```diff
@@ -76,232 +71,286 @@
...
+    color: $black-87;
   }
 
   &.business {
-    background-color: $business-bg;
-
-    h2 {
-      color: $business;
-    }
+    background-color: $mustard;
```
Comment (fernandolins) [inline themes/medius/scss/home.scss:247]:
```suggestion
    background-color: $mustard-40;
```

File: themes/medius/scss/home.scss [hunk 2]
```diff
@@ -76,232 +71,286 @@
...
   }
 
   &.conferences {
-    background-color: $conferences-bg;
-
-    h2 {
-      color: $conferences;
-    }
+    background-color: $teal;
```
Comment (fernandolins) [inline themes/medius/scss/home.scss:251]:
```suggestion
    background-color: $teal-40;
```

File: themes/medius/scss/home.scss [hunk 3]
```diff
@@ -76,232 +71,286 @@
...
   }
 
   &.design {
-    background-color: $design-bg;
-
-    h2 {
-      color: $dark-purple;
-    }
+    background-color: $coral;
```
Comment (fernandolins) [inline themes/medius/scss/home.scss:255]:
```suggestion
    background-color: $coral-40;
```

File: themes/medius/scss/home.scss [hunk 4]
```diff
@@ -76,232 +71,286 @@
...
   }
 
   &.development {
-    background-color: $development-bg;
+    background-color: $purple;
```
Comment (fernandolins) [inline themes/medius/scss/home.scss:259]:
```suggestion
    background-color: $purple-60;
```

**Review by fernandolins — CHANGES_REQUESTED** [review]

File: themes/medius/scss/home.scss (hunk 1 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:247]: There are variables to inserting the default shade for each color stack. Are you proposing we dont use those variables?

File: themes/medius/scss/home.scss (hunk 2 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:251]: There are variables to inserting the default shade for each color stack. Are you proposing we dont use those variables?

File: themes/medius/scss/home.scss (hunk 3 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:255]: There are variables to inserting the default shade for each color stack. Are you proposing we dont use those variables?

File: themes/medius/scss/home.scss (hunk 4 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:259]: There are variables to inserting the default shade for each color stack. Are you proposing we dont use those variables?

File: themes/medius/scss/home.scss (hunk 1 above)
Comment (fernandolins) [reply themes/medius/scss/home.scss:231]: Yes, if we have created the new variables to be replaced the old ones.
Why shouldn't we use the new ones?

**Review by fernandolins — CHANGES_REQUESTED** [review]

File: themes/medius/scss/home.scss (hunk 1 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:231]: This is a new one. At least it is how it is been used on the DS. We have the number code for each stack position and a general variable, like the one above, $mustard, to load the default value to the mustard color stack

**Pull request merged**